server/activity.go

```diff
@@ -76,3 +80,31 @@ func addActivity(db *gorm.DB, userId uint, ar ActivityAddRequest) (Activity, err
 	slog.Debug("Adding activity", "added_activity", activity)
 	return activity, nil
 }
+
+func updateActivity(db *gorm.DB, id uint, activityUpdateRequest ActivityUpdateRequest) error {
```

Comment:
Is there anything in place to ensure an authenticated user can only update their own activity?

Reply:
Good point, I'm still getting my bearings. Will update.